src/core_plugins/kibana/public/dashboard/partials/save_dashboard.html

```diff
@@ -18,5 +18,7 @@
       <kbn-info info="Change the time filter to the currently selected time each time this dashboard is loaded"></kbn-info>
     </label>
   </div>
-  <button type="submit" ng-disabled="!opts.dashboard.title" class="btn btn-primary" aria-label="Save dashboard">Save</button>
+  <button type="submit" ng-disabled="!opts.dashboard.title" class="btn btn-primary" aria-label="Save dashboard">
+    {{opts.dashboard.titleChanged() ? 'Save a Copy' : 'Save'}}
```

Comment:
I tested the UX of this, and I think that by making the title editable immediately, we are making it feel like the user can rename the dashboard. When the user then starts changing the name, the button changes to "Save a Copy". I feel like there's a high possibility that this will confuse users: "Why did that button just change? Wait, did it really change or was it there all along? What did I do to cause it to change?"

In this situation, I think we really need to be careful to put the user in full control of the app. This means the UI should change in reaction to very explicit commands from the user. In this case, the user's explicit command was to change the name of the dashboard -- not to disable "Save" and enable "Save As". Even though it makes sense to us and the application internally, the rationale is opaque to the user.

I think we should use the Courté Checkbox here. When the user clicks the checkbox, s/he is very explicitly commanding the app to save a new document, so a change in the UI in response to this command makes sense.